### PR TITLE
Fix input to chown_path

### PR DESCRIPTION
**Pull request by convolu**

### CODE REVIEW

**Comment by convolu** [comment]
@pazdera @tombettany I noticed this issue by accident, could you please review?

**Comment by pazdera** [comment]
Looks good :shipit:!

File: kano_profile/tracker.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def session_end(session_file):
                 with wf:
                     json.dump(data, wf)
                 if 'SUDO_USER' in os.environ:
-                    chown_path(data)
+                    chown_path(session_file)
```
Comment (tombettany) [inline kano_profile/tracker.py]: Is it not a problem trying to set the owner with the file already opened, is there not potential for a later write overwriting these permissions?

**Comment by convolu** [comment]
@tombettany @pazdera  I have updated this to attempt to do the chown after the file has been closed, thanks for catching that.

**Comment by convolu** [comment]
Merging to test with Jessie

**Pull request merged**